apps/coscrad-frontend/src/components/tenant/tenant.tsx

```diff
@@ -0,0 +1,16 @@
+import { Stack } from '@mui/material';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { Copyright } from '../copyright/copyright';
+import { TenantLogo } from './tenant-logo';
+
+export const Tenant = (): JSX.Element => {
+    const { copyrightHolder } = useContext(ConfigurableContentContext);
```

Comment:
You should use context to get the state from the content config in the page (component that has a custom-route registered). These pages are non-index-to-detail flow pages, what we used to call "front matter".

You should probably have a separate presenter that takes this as part of the props, but if the page's component is small, it's not really giving us much value to break that out. Keep in mind that most of these will disappear once we move to the
`coscrad-front-end` lib => standalone React projects for builds
approach, and later the CMS approach.

Reply:
So in this case, the flow goes `Footer` -> `Tenant` -> `TenantLogo`.  Should I limit the call to `useContext` to the `Footer` and then just prop-drill the `copyrightHolder` down from there?  Or does it matter?  It seems cleaner to just call `useContext` here because the drilling seems awkward.

Reply:
I'm ok with using the context inside the `Copyright` component instead. If that's the case, make sure there is absolutely zero prop drilling going on in `Tenant` and use context for every component. Tenant just collects stateful components together, much like the main <App> does.